api/controllers/ClientController/register.ts

```diff
@@ -10,11 +10,11 @@ async function register(req: RedstoneRequest, res: RedstoneResponse) {
 
   if (!data.username || !data.email || !data.password || !data.nickname) {
     return res.status(400).json({
-      message: '缺少参数：username，email，nickname 或 password。',
+      message: res.__('Lack_arguments') + ': ' + 'username, email, nickname, password.',
```

Comment:
Try using string template notations to simplify this code.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals